vm/capi/kernel.cpp

```diff
@@ -60,8 +60,7 @@ extern "C" {
       ep.pop(env);
 
       if(env->state()->thread_state()->raise_reason() != cException) {
```

Comment:
I guess it would be safer to check for `cBreak` explicitly and only then return the break value instead of doing it for all kinds of exceptions?

I think a switch() is reasonable here.